src/main/java/com/ibm/cloud/cloudant/kafka/connect/CloudantSinkTask.java

```diff
@@ -123,4 +167,10 @@ public void close(Collection<TopicPartition> partitions) {
 		LOG.info("Committed ");
 	}
 
+	// for testing
+	// TODO find out if there is a better way to do this without changing production classes
+	public void setContext(SinkTaskContext ctx) {
+		this.context = ctx;
+	}
```

Comment:
The `SinkTask` API appears to offer [`initialize(SinkTaskContext)`](https://kafka.apache.org/31/javadoc/org/apache/kafka/connect/sink/SinkTask.html#initialize(org.apache.kafka.connect.sink.SinkTaskContext))

Reply:
Thanks for spotting this - done in 6a8928b